resolwe/process/parser.py

```diff
@@ -173,6 +173,12 @@ def visit_ClassDef(self, node):
             else:
                 continue
 
+            # Value ``None`` can occur if there is another top level class
+            # definition in the module that does not inherit from classes
+            # defined in ``runtime``.
+            if base is None:
```

Comment:
It would make more sense to put this condition in the if below:
```
if base is not None and issubclass(base, runtime.Process):
    break
```
Or to make it even more robust:
```
from inspect import isclass

if isclass(base) and issubclass(base, runtime.Process):
    break
```
This is also more self-explanatory, so you can also remove the comment.

Reply:
Good point. Fixed.